tests/acceptance/async_adapter_tests.py

```diff
@@ -612,7 +612,7 @@ def begin(self, channel):
         timer1 = channel.connection.add_timeout(0, self.on_my_timer)
         timer2 = channel.connection.add_timeout(0, self.on_my_timer)
 
-        self.assertNotEqual(timer1, timer2)
```

Comment:
Were you seeing this fail too?

Reply:
Yes, since the timer class implements value equality this would evaluate to equal as both timers have `0` as their deadline.